src/icons/microsoft-azure.svg

```diff
@@ -0,0 +1,5 @@
+<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+<path d="M0.57735 13C0.220084 12.3812 0.220085 11.6188 0.57735 11L5.42265 2.6077C5.77992 1.98889 6.44017 1.6077 7.1547 1.6077L16.8453 1.6077C17.5598 1.6077 18.2201 1.98889 18.5774 2.6077L23.4226 11C23.7799 11.6188 23.7799 12.3812 23.4226 13L18.5774 21.3923C18.2201 22.0111 17.5598 22.3923 16.8453 22.3923L7.1547 22.3923C6.44017 22.3923 5.77992 22.0111 5.42265 21.3923L0.57735 13Z" fill="#4387FB"/>
+<path d="M18.1415 17C15.1342 17 12.1268 17 9.11951 17C10.9239 16.5926 12.7283 16.287 14.5327 15.8796C14.5327 15.7778 14.5327 15.7778 14.4468 15.7778C13.5016 14.3519 12.5565 13.0278 11.6972 11.8056C11.6113 11.7037 11.6113 11.6019 11.6972 11.6019C12.0409 10.4815 12.3846 9.463 12.6424 8.34263C12.8142 7.83338 12.9861 7.32412 13.072 6.81486C13.072 6.81486 13.072 6.71301 13.1579 6.71301C14.7905 10.176 16.423 13.5371 18.1415 17Z" fill="white"/>
+<path d="M12.7281 6C12.5563 6.40741 12.3844 6.71296 12.2985 7.12037C11.4393 9.25924 10.666 11.2963 9.80672 13.4351C9.54895 14.1481 9.20525 14.8611 8.94748 15.6759C8.94748 15.7777 8.86156 15.7777 8.77563 15.7777C7.9164 15.7777 7.14308 15.7777 6.28384 15.7777C6.112 15.7777 6.112 15.7777 6.19792 15.6759C7.14308 13.6389 8.17417 11.7037 9.11933 9.66665C9.11933 9.5648 9.20525 9.46295 9.29118 9.3611C10.4082 8.34258 11.5252 7.22222 12.6422 6.10185C12.6422 6.10185 12.6422 6 12.7281 6Z" fill="white"/>
+</svg>
```

Comment:
Do we use minification for svg icons?
like svgo @abhinavrastogi-harness

Reply:
let use svgo for this icon

Reply:
done, thanks! :)